easybuild/easyconfigs/p/pysteps/pysteps-1.7.1-foss-2022a.eb

```
@@ -0,0 +1,52 @@
+easyblock = 'PythonBundle'
+
+name = 'pysteps'
+version = '1.7.1'
+
+homepage = 'https://pysteps.github.io/'
+description = """
+Pysteps is an open-source and community-driven Python library for probabilistic
+precipitation nowcasting, i.e. short-term ensemble prediction systems."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+    ('OpenCV', '4.6.0', '-contrib'),
+    ('Pillow', '9.1.1'),
+    ('pyproj', '3.4.0'),
+    ('matplotlib', '3.5.2'),
+    ('dask', '2022.10.0'),
+    ('netcdf4-python', '1.6.1'),
+    ('xarray', '2022.6.0'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('jsmin', '3.0.1', {
+        'checksums': ['c0959a121ef94542e807a674142606f7e90214a2b3d1eb17300244bbb5cc2bfc'],
+    }),
+    ('pyFFTW', '0.13.0', {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'checksums': ['da85102405c0bd95d57eb19e99b01a0729d8406cb204c3900894b873784253da'],
+    }),
+    (name, version, {
+        'patches': ['pysteps-1.7.1_h5py-3.7_compat.patch'],
+        'source_tmpl': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/pySTEPS/pysteps/archive/'],
+        'checksums': [
+            {'v1.7.1.tar.gz': '5063d7546d9995ec6ca52a5b4e50ef3f3c1ae43088a55e370af4aec18162a22f'},
+            {'pysteps-1.7.1_h5py-3.7_compat.patch': 'c2b1a47d7151a7f4072dce797c6303314c0d8fd7683b0001deb43f24357ed977'},
+        ],
+    }),
+    ('pysteps-nwp-importers', '6ca1582', {
+        'source_tmpl': '%(version)s.tar.gz',
```

@smoors Please rename on download, and use a better version (datestamp)

```suggestion
    ('pysteps-nwp-importers', '20220124', {
        'sources': [{'download_filename': '6ca1582.tar.gz', 'filename': SOURCE_TAR_GZ}],